### DISCUSSION TITLE
Confused between plugin-syntax-dynamic-import and plugin-proposal-dynamic-import

### DISCUSSION THREAD

**Discussion by brandonburkett** [babel/babel#11804] [Q&A] [answered]
Hello,

I use dynamic imports for lazy loading vue components and was upgrading from babel 7.7.4 to 7.10.4.

We were using `@babel/plugin-syntax-dynamic-import` but I see this is only at 7.8.3?  Is this right? Is there no 7.10.4 version?  Is this being replaced by `@babel/plugin-proposal-dynamic-import` which is at 7.10.4?

Thanks!

**Comment by existentialism** [comment, marked as answer]
@brandonburkett the syntax-* plugins enable the syntax within our parser. The latest version for `@babel/plugin-syntax-dynamic-import` is correct as we only increment package versions when we release for packages that _have changes_.

`@babel/plugin-proposal-dynamic-import` is not necessary if you are using a bundler like Webpack or Rollup.

**Comment by brandonburkett** [comment]
Thank you so much for confirming @existentialism
